ecommerce/serializers.py

```diff
@@ -439,6 +439,9 @@ def validate_items(self, items):
                 product__id=product_id
             ).exists():
                 raise ValidationError(f"Invalid product id {product_id}")
+            product = models.Product.all_objects.filter(id=product_id).first()
```

Comment:
Can we not do something like:
```
try:
  product = models.Product.objects.get(id=product_id)
except models.Product.DoesNotExist:
  raise ValidationError(f"Product id {product_id} is not active")
```

Reply:
It is portraying the object does not exist rather then revealing the active/in-active state.